pages/linux/xbps.md

```diff
@@ -1,6 +1,6 @@
 # xbps
 
-> The X Binary Package System (or xbps) is the binary package system used by Void Linux.
+> The X Binary Package System (or xbps) is the binary package system used by Void Linux. More information: https://github.com/void-linux/xbps.
```

Comment:
Perhaps this would be more consistent with other pages?
```suggestion
> The X Binary Package System (or xbps) is the binary package system used by Void Linux.
> More information: <https://github.com/void-linux/xbps>.
```